src/test/java/ioananadroflorea/tests/Tema5/DemoQA.java

```diff
@@ -0,0 +1,111 @@
+package ioananadroflorea.tests.Tema5;
+
+import ioanaandroflorea.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.Keys;
+
+public class DemoQA {
+    public static void main(String[] args) {
+        textBox();
+        checkBox();
+        radioButton();
+        webTables();
+        button();
+    }
+
+    public static void textBox(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement textBox = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span"));
```

Comment:
When using xpath as a selector try to avoid using absolute xpath(meaning that you start from the html tag) and use relative xpath(meaning that you'll try to minimize the xpath by starting with the closest unique parent and continuing from there) instead. For example "/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span" would become "//ul[@Class='menu-list']/li[1]/span[text()='Text Box']"

Reply:
class is not with capital letter there. it seems that there is a user with this username

Reply:
You mean that is not DemoQA and should be demoqa?

Reply:
No, I was not referring to the name of the class but to -> @class from the first comment on this thread